DXRModules/DeusEx/Classes/DXRFlags.uc

```diff
@@ -1276,20 +1341,34 @@ simulated function ExecMaxRando()
     settings.medbotamount = int(rngb()) + 1;
     settings.repairbotamount = int(rngb()) + 1;
 
-    settings.doorsmode = undefeatabledoors + doorindependent;
-    settings.doorsdestructible = rng(100);
-    settings.doorspickable = rng(100);
+    if (forceMenuOptions){
+        //Eventually we can add logic to randomize between the door menu options
+    } else {
+        settings.doorsmode = undefeatabledoors + doorindependent;
+        settings.doorsdestructible = rng(100);
+        settings.doorspickable = rng(100);
+    }
+
+    /* To match the menu options, we just randomize between 0, 50, and 100 */
+    if (forceMenuOptions){
+        settings.deviceshackable = rng(3)*50;
+    } else {
+        settings.deviceshackable = rng(100);
+    }
 
-    settings.deviceshackable = rng(100);
     MaxRandoVal(settings.enemiesrandomized);
-    settings.hiddenenemiesrandomized = settings.enemiesrandomized;
```

Comment:
why did you remove the randomization of `hiddenenemiesrandomized `?

Reply:
That's weird - definitely not intentional.